node/pkg/governor/governor_test.go

```diff
@@ -959,3 +961,311 @@ func TestTestnetConfigIsValid(t *testing.T) {
 	err := gov.initConfig()
 	require.NoError(t, err)
 }
+
+func TestLargeTransactionGetsEnqueuedAndReleasedWhenTheTimerExpires(t *testing.T) {
```

Comment:
I just wanted to comment that I like the full-flow explanation here, such cases should make it easier in the future to copy/paste and express complex PoC scenarios which should be very testable, thanks!